modules/crio_trouble.adoc

```diff
@@ -0,0 +1,431 @@
+  docker.io/openshift/prometheus-alertmanager    v0.15.2 68bbd00063784 242MB
+  docker.io/openshift/prometheus-node-exporter   v0.16.0 f9f775bf6d0ef 225MB
+  quay.io/coreos/cluster-monitoring-operator     v0.1.1  4488a207a5bca 531MB
+  quay.io/coreos/configmap-reload                v0.0.1  3129a2ca29d75 4.79MB
+  quay.io/coreos/kube-rbac-proxy                 v0.3.1  992ac1a5e7c79 40.4MB
+  quay.io/coreos/kube-state-metrics              v1.3.1  a9c8f313b7aad 22.2MB
+```
+To see the pods that are currently active in the CRI-O environment, use
+`crictl pods`:
+
+```
+$ sudo crictl pods
+
+POD ID        CREATED     STATE  NAME                   NAMESPACE              ATTEMPT
+09997515d7729 5 hours ago Ready  kube-state-metrics-... openshift-monitoring   0
+958b0789e0552 5 hours ago Ready  node-exporter-rkbzp    openshift-monitoring   0
+4ec0498dacec8 5 hours ago Ready  alertmanager-main-0    openshift-monitoring   0
+2873b697df1d2 5 hours ago Ready  cluster-monitoring-... openshift-monitoring   0
+b9e221481fb7e 5 hours ago Ready  router-1-968t4         default                0
+f02ce4a4b4186 5 hours ago Ready  sdn-c45cm              openshift-sdn          0
+bdf5b1dcc0a08 5 hours ago Ready  ovs-kdvzs              openshift-sdn          0
+49dbc57455c8f 5 hours ago Ready  sync-hgfvb             openshift-node         0
+```
+
+To see containers that are currently running, run the `crictl ps` command:
+```
+$ sudo crictl ps
+CONTAINER ID  IMAGE                                CREATED     STATE   NAME                 ATTEMPT
+376eb13e3cb37 quay.io/coreos/kube-state-metrics... 4 hours ago Running kube-state-metrics   0
+72d61c3d393b5 992ac1a5e7c79d627321dc7877f741a00... 4 hours ago Running kube-rbac-proxy-self 0
+5fa8c93484055 992ac1a5e7c79d627321dc7877f741a00... 4 hours ago Running kube-rbac-proxy-main 0
+a2d35508fc0ee quay.io/coreos/kube-rbac-proxy...    4 hours ago Running kube-rbac-proxy      0
+9adda43f3595f docker.io/openshift/prometheus-no... 4 hours ago Running node-exporter        0
+7f4ce5b25cfdb docker.io/openshift/oauth-proxy...   4 hours ago Running alertmanager-proxy   0
+85418badbf6ae quay.io/coreos/configmap-reload...   4 hours ago Running config-reloader      0
+756f20138381c docker.io/openshift/prometheus-al... 4 hours ago Running alertmanager         0
+5e6d8ff4852ba quay.io/coreos/cluster-monitoring... 4 hours ago Running cluster-monitoring-  0
+1c96cfcfa10a7 docker.io/openshift/origin-haprox... 5 hours ago Running route                0
+8f90bb4cded60 docker.io/openshift/origin-node...   5 hours ago Running sdn                  0
+59e5fb8514262 docker.io/openshift/origin-node...   5 hours ago Running openvswitch          0
+73323a2c26abe docker.io/openshift/origin-node...   5 hours ago Running sync                 0
+```
+To see both running containers as well as containers that are stopped or
+exited, use `crictl ps -a`:
+```
+$ sudo crictl ps -a
+```
+
+If your CRI-O service is stopped or otherwise not working, you can list the
```

Comment:
s/ or otherwise not working/

Reply:
I changed it as follows: s/ or otherwise not working/ or malfunctioning/
That covers the case where the container might still be running, but having some problems.